.github/pull_request_template.md

```diff
@@ -1,20 +1,22 @@
-### 💬 Describe the pull request
-A clear and concise description of what the pull request is about.
+### 💬 Describe the pull request/code changes
+_A clear and concise description of what the pull request is about._
 
-### 🗃️ Issues
-This pull request is related to :
-- #1
-- #2
+### 🎟️ Jira
+_This pull request is related to the following Jira issue(s):_
 
 ### 🔢 To Review
-Steps to review the PR:
-1. Install '...'
-2. Go to '...'
-3. Click on '....'
-4. Scroll down to '....'
+_Steps to review the PR:_
+1. Add some details for the reviewer here.
 
-### 🔖 Does it need an entry in the CHANGELOG ?
+### 🔖 Does it need an entry in the CHANGELOG?
  - [ ] Update the "NEXT RELEASE" section of the `CHANGELOG.md`
 
 ### 📸 Screenshots
-If applicable, add screenshots to help explain the task.
+_If applicable, add screenshots to help explain the task._
+
+### 📦 Ready to merge
+- [ ] The reviewer approves the pull request.
```

Comment:
We already have this information with GitHub. What is the reason to duplicate it?